tools/ci/taskcluster-run.py

```diff
@@ -42,7 +42,7 @@ def gzip_file(filename, delete_original=True):
         os.unlink(filename)
 
 
-def main(product, commit_range, wpt_args):
+def main(product, commit_range, py3, wpt_args):
```

Comment:
Right now this won't do anything I think, because you didnt update the callsite (`tools/ci/tc/tasks/test.yml`).

That said, this is just for trigger runs, so we can just hard-code `py3` to be True in this PR! :)

Reply:
You are right :). Code updated.